admob/integration_test/src/integration_test.cc

```diff
@@ -287,6 +287,14 @@ TEST_F(FirebaseAdMobTest, TestInitializationStatus) {
       << "Expected adapter class '" << kAdMobClassName << "' is not loaded.";
 }
 
+TEST_F(FirebaseAdMobTest, TestDisableSDKCrashReporting) {
+  firebase::admob::DisableSDKCrashReporting();
+}
+
+TEST_F(FirebaseAdMobTest, TestDisableMediationInitialization) {
+  firebase::admob::DisableMediationInitialization();
```

Comment:
It seems like these tests aren't really testing the functionality, since it needs to be called before initialization.  I think a comment regarding what this is actually testing (that calling the function won't have issues, and not necessarily the functionality) would be good.

Reply:
Done!

Reply:
You could also implement a second test fixture class that doesn't automatically initialize AdMob to test what happens when you call these before initialization.